sdk/src/client/node_api/core/routes.rs

```diff
@@ -91,6 +91,18 @@ impl ClientInner {
             .await
     }
 
+    /// Checks if the account is ready to issue a block.
```

Comment:
TIP writes Check, not Checks 🏃‍♂️

Reply:
It's fine, we don't alway keep 100% like the TIPs